### PR TITLE
Updating terraform and k8s files adding redis...

### PR DESCRIPTION
**Pull request by LumosViridi**
Also updated the way secrets are generated with Terraform and some code cleanup

### CODE REVIEW

**Comment by LumosViridi** [comment]
@charlesBochet / @FelixMalfait Hey! Took me a while to get this updated but this should include all of the changes needed up through `v0.30.0` for storage, redis, pg-boss, and the worker 😊

Do you mind taking a look when you have a min?

**Comment by FelixMalfait** [comment]
Oh wow, thanks @LumosViridi!

File: packages/twenty-docker/k8s/manifests/deployment-redis.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          env:
+            - name: PORT
+              value: 6379
+          ports:
+            - containerPort: 6379
+              name: redis
+              protocol: TCP
+          resources:
+            requests:
+              memory: "1024Mi"
```
Comment (charlesBochet) [inline packages/twenty-docker/k8s/manifests/deployment-redis.yaml:37]: looks well sized

File: packages/twenty-docker/k8s/manifests/deployment-server.yaml [hunk 2]
```diff
@@ -22,67 +22,80 @@ spec:
...
+          image: twentycrm/twenty:latest
+          imagePullPolicy: Always
+          env:
+            - name: PORT
+              value: 3000
+            - name: SERVER_URL
+              value: "https://crm.example.com:443"
+            - name: FRONT_BASE_URL
+              value: "https://crm.example.com:443"
+            - name: "BACKEND_SERVER_URL"
```
Comment (charlesBochet) [inline packages/twenty-docker/k8s/manifests/deployment-server.yaml:42]: is this one useful?

File: packages/twenty-docker/k8s/manifests/deployment-worker.yaml [hunk 3]
```diff
@@ -21,58 +21,60 @@ spec:
...
+              valueFrom:
+                secretKeyRef:
+                  name: tokens
+                  key: fileToken
+          command:
+            - yarn
+            - worker:prod
+          resources:
+            requests:
+              memory: "256Mi"
```
Comment (charlesBochet) [inline packages/twenty-docker/k8s/manifests/deployment-worker.yaml:72]: likely to be too small for messaging scripts. ~1GB should be better

File: packages/twenty-docker/k8s/manifests/pv-docker-data.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: twentycrm-docker-data-pv
+spec:
+  storageClassName: default
+  capacity:
+    storage: 10Gi
```
Comment (charlesBochet) [inline packages/twenty-docker/k8s/manifests/pv-docker-data.yaml:8]: we only store a flag file in this volume so 10Gi looks overkill

**Review by charlesBochet — APPROVED** [review]
Thank you, this is awesome!

I have left a few comments but approving as it LGTM :)

File: packages/twenty-docker/k8s/manifests/deployment-server.yaml (hunk 2 above)
Comment (LumosViridi) [reply packages/twenty-docker/k8s/manifests/deployment-server.yaml:42]: This wasn't defined in the docker compose file, so I'll remove it!

**Comment by charlesBochet** [comment]
Merging it, thank you!

**Pull request merged**